environments/update_chs_configs.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
```

Comment:
I think it's worth getting the detail on your initial PR comment into a comment block at the top of this shell script - explaining what it's for and how it's used. Probably also worth then referencing this script in the promise to file README.MD file (which I don't think exists yet).

Reply:
Added comment.